eng/ci/official-build.yml

```diff
@@ -0,0 +1,63 @@
+name: $(Build.SourceBranchName)_$(Build.Reason)_$(BuildNumber)
+
+trigger:
+  batch: true
+  branches:
+    include:
+    - main
+    - release/*
+    - v3.x
+
+schedules:
+  # Ensure we build nightly to catch any new CVEs and report SDL often.
+  - cron: "0 0 * * *"
+    displayName: Nightly Build
+    branches:
+      include:
+      - main
+      - release/*
+      - v3.x
+    always: true
+
+# CI only, does not trigger on PRs.
+pr: none
+
+resources:
+  repositories:
+  - repository: 1es
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
+  - repository: eng
+    type: git
+    name: engineering
+    ref: refs/tags/release
+
+variables:
+  - template: ci/variables/build.yml@eng
+  - template: ci/variables/cfs.yml@eng
+  - template: /eng/ci/templates/variables/build.yml@self
+
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1es
+  parameters:
+    pool:
+      name: 1es-pool-azfunc
+      image: 1es-windows-2022
+      os: windows
+
+    stages:
+    - stage: Initialize
+      jobs:
+      - template: /eng/ci/templates/jobs/install-dotnet.yml@self
```

Comment:
Jobs are potentially invoked on different agents, so installing dotnet here does **not** install it for the rest of the jobs. You will need to use dotnet installation per job as a step.